### PR TITLE
quote icon fixes

### PR DESCRIPTION
**Pull request by texas-kristen**
Proposed fix for #1649 

- adds end quote icon
- updates color to use actual hex code so that the icons are blue (component isn't finding the alias assigned in tailwind.config.js)
- adjusts width/margin/padding so that end quote icon will render at the end of the quote, not on a new line
- lightens background around quote for emphasis

**Alternative**

I like the contrast around the quotes, but this is how it looks without it, let my know whatcha prefer

image

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 6e20a0801b59167ed68de34fa401f61dfe467425

https://deploy-preview-1651--stoplightio.netlify.app

**Comment by texas-kristen** [comment]
To preview changes, click to https://deploy-preview-1651--stoplightio.netlify.app/demo/ directly instead of using Demo button in preview

File: src/components/Hero/LandingHero.tsx [hunk 1]
```diff
@@ -118,12 +118,13 @@ export const LandingHero: React.FunctionComponent<ILandingHero> = ({
...
-                            <Icon icon="quote-left" size="lg" style={{ color: 'blue2' }} />
-                            <div className="flex flex-col ml-8">
-                              <h3 className="-mt-5 ">{item.summary}</h3>
-                              <div className="mt-2">— {item.name}</div>
+                        <div className="max-w-full mb-10 mr-32 px-3 py-3 bg-lighten-600 rounded-xl" key={i}>
+                        <div className="flex flex-col sm:mx-10">
+                          <Icon icon="quote-left" className="pr-2" size="lg" style={{ color: '#0b6fcc' }} />
+                          <div className="flex flex-col ml-8">
+                            <h3 className="-mt-5 ">{item.summary}
+                            <Icon icon="quote-right" className="pl-2" size="lg" style={{ color: '#0b6fcc' }} /></h3>
```
Comment (paulatulis) [inline src/components/Hero/LandingHero.tsx:126]:
```suggestion
                            <Icon icon="quote-right" className="pl-2" size="lg" className="text-blue2" /></h3>                          
```

Comment (paulatulis) [reply src/components/Hero/LandingHero.tsx:126]: You should be able to get the color to show up this way

File: src/components/Hero/LandingHero.tsx [hunk 2]
```diff
@@ -118,12 +118,13 @@ export const LandingHero: React.FunctionComponent<ILandingHero> = ({
...
                       {list?.items?.map((item, i) => (
-                        <div className="mb-10 mr-48" key={i}>
-                          <div className="flex flex-col sm:mx-10">
-                            <Icon icon="quote-left" size="lg" style={{ color: 'blue2' }} />
-                            <div className="flex flex-col ml-8">
-                              <h3 className="-mt-5 ">{item.summary}</h3>
-                              <div className="mt-2">— {item.name}</div>
+                        <div className="max-w-full mb-10 mr-32 px-3 py-3 bg-lighten-600 rounded-xl" key={i}>
+                        <div className="flex flex-col sm:mx-10">
+                          <Icon icon="quote-left" className="pr-2" size="lg" style={{ color: '#0b6fcc' }} />
```
Comment (paulatulis) [inline src/components/Hero/LandingHero.tsx:123]:
```suggestion
                          <Icon icon="quote-left" className="pr-2" size="lg" className="text-blue2" />
```

**Review by paulatulis — APPROVED** [review]
Looks good to me!

**Pull request merged**